mlrun/runtimes/base.py

```diff
@@ -692,8 +692,10 @@ def _enrich_run(
                 if separator in short_name:
                     short_name = short_name.split(separator)[-1]
             def_name += "-" + short_name
+        # most of the times handlers will have `_` in the handler name (python convention is to separate function words
+        # with `_`), therefore we don't want to be noisy when normalizing the run name
         runspec.metadata.name = normalize_name(
-            name or runspec.metadata.name or def_name
+            name or runspec.metadata.name or def_name, verbose=False
```

Comment:
IIUC we only want the disable verbosity when using `def_name`. I think we should condition the verbosity by that.